### ISSUE TITLE
Move LSR budgetary impact to Budgetary impact screen

### ISSUE THREAD

**Issue by MaxGhenis** [PolicyEngine/policyengine-app#1826]
The budgetary impact subtitle breaks up the title and the chart. It's also inconsistent, as it's only on the earnings chart and only shows the effect on budgetary impacts. I'd suggest we instead conditionally display it in the budgetary impact chart, as a note below the chart, if any elasticities are nonzero (or if easier to start, if LSR has a nonzero effect).

**Comment by anth-volk** [comment]
It seems this is superseded by #1829?

**Comment by MaxGhenis** [comment]
Yes but let's return to it after figuring out how to best describe it given federal/state differences

**Comment by MaxGhenis** [comment]
Thank you for filing this issue. Closing as part of the `policyengine-app` (v1) wind-down: policyengine.org is now served by [`policyengine-app-v2`](https://github.com/PolicyEngine/policyengine-app-v2), and this repository is being archived, so this issue is frozen against a codebase that no longer deploys anywhere.

If this is still relevant on the current site, please re-file it against [`policyengine-app-v2`](https://github.com/PolicyEngine/policyengine-app-v2) — we'd welcome it there. Apologies that this sat open through the migration.
